features/FxA-51-connect-another-device/README.md

```diff
@@ -0,0 +1,130 @@
+* Start date of test:
+* End date of test:  
+* Feature card: https://waffle.io/mozilla/fxa-features/cards/5820a9739ca711d4005a5408
+* Other:
+  * This feature document is the result of our work week in Toronto to increase multi-device adoption from first run sign-up. The email confirmation we envisioned during the workweek will be divided into 4 phases
+    * Phase 2: https://docs.google.com/document/d/133_Cu0TlWQZIw-ihMH1s-U_80I12MXZcvHQEuFhWbPg/edit#heading=h.qrbb2drvq5dg (WIP)
+  * Proposed new first run user flow in Invision: https://mozilla.invisionapp.com/share/ZX8RK05G7#/screens/191829453
+  * Graph of multi-device adoption within 48h and within 7 days: https://sql.telemetry.mozilla.org/queries/1254#2173
+  * Single vs multi-device MAU: https://sql.telemetry.mozilla.org/queries/256#486
+
+## Problem Summary and Context
+Sync setup through first run via FxA serves the purpose of helping users setup multiple devices and to ensure their data is syncing across their devices.
+
+Unfortunately, within the first 48h of registering for Sync, only 6.4% of users setup a second device and approximately only 8% after 7 days (as of October 30th).
+
+Additionally, approximately only 15% of FxA MAU use their account on two different devices. This value is dropping every month as the number of single device users keep growing. (as of September 30th)
+
+We observed that our sign-up flow does not detect UA to recommend appropriate next steps to complete setup of two devices.
+
+We also observed that the signup flow mostly ends on confirming the user’s email. Mobile download is only very passively promoted.
+
+We need to better guide the user through the setup of two devices while removing as much friction as possible. Next steps should always be clear to users.
+
+### Assumptions
+We assume that users signing up for Sync have an understanding of why they are setting up an account so we should merely be facilitating setup after email confirmation.
+
+(edit: we have observed through user testing that new users do not understand the first run page very well. This will be the subject of one of our next tests.)
+
+## Outcomes
+The outcome we are looking for is a slick user flow that makes it obvious to users that in order to complete their setup of Sync, they need to add another device after email confirmation.
+
+## Hypothesis
+If we refine our UI/UX during sign up to remove friction around setting up a second device for firstrun users on desktop, then we will see an increase in multi-device users within the first 7 days of registration because we have observed that less than 30% of registration (65k clicks / 210k registration / month) engage with our mobile download buttons and at best 26% (8% 7d md / 30% ctr) of them follow through to an install within 7 days after clicking.
+
+## Metrics
+We will know our hypothesis to be true when we see the proportion of multi-device users increase after 48h and 7d of registration to FxA+Sync on firstrun.
```

Comment:
`registration to FxA+Sync on firstrun.` is it only firstrun? or would we also look at the 'normal' sign in flow via the FF Desktop menu?